### PR TITLE
Update the RHUI TSG.

### PR DESCRIPTION
**Pull request by navpreetkaur1098**
# Pull request guidance

There is a small change that is required in the RHUI Troubleshooting guide as this is directly used by azure customers to update their RHEL VMs.

### CODE REVIEW

**Review by msaenzbo — CHANGES_REQUESTED** [review]
Why you are requesting the change, the same enable Microsoft* is used for expired certificate.

**Comment by navpreetkaur1098** [comment]
> Why you are requesting the change, the same enable Microsoft* is used for expired certificate.

This command "sudo yum reinstall $(rpm -qa | grep -i rhui-azure) --disablerepo=* --enablerepo="*microsoft*"" is updating all the packages which include microsoft keyword. Packages from Packages.Microsoft.com are also getting updated in the customers VMs. This is not the expectation. Only a specific package is supposed to be updated. Therefore, the changes are introduced.

**Review by divargas-msft — CHANGES_REQUESTED** [review]
The line that needs to be updated is 53:

```bash
sudo yum update -y --disablerepo='*' --enablerepo='*microsoft-azure*'
```

There’s no harm in the reinstall command modification, you can keep that one as well.

Once line 53 is modified, we are good with this one.

File: support/azure/virtual-machines/linux/troubleshoot-linux-rhui-certificate-issues.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ All the commands in the following steps should be run by using root privileges o
 3. Reinstall the corresponding `rhui-azure` package by running the `yum reinstall` command:
 
    ```bash
-   sudo yum reinstall $(rpm -qa | grep -i rhui-azure) --disablerepo=* --enablerepo="*microsoft-azure*"
+   sudo yum reinstall $(rpm -qa | grep -i rhui-azure) --disablerepo=* --enablerepo="*microsoft*"
```
Comment (divargas-msft) [inline support/azure/virtual-machines/linux/troubleshoot-linux-rhui-certificate-issues.md:113]: @AmandaAZ please keep this line using microsoft-azure as well, as the PR creator proposed. That's OK

**Review by divargas-msft — CHANGES_REQUESTED** [review]
Please keep microsoft-azure in the reinstall command too

File: support/azure/virtual-machines/linux/troubleshoot-linux-rhui-certificate-issues.md (hunk 1 above)
Comment (AmandaAZ) [reply support/azure/virtual-machines/linux/troubleshoot-linux-rhui-certificate-issues.md:113]: OK. I will create a new PR to update it.

**Comment by AmandaAZ** [comment]
@navpreetkaur1098 @divargas-msft The line 113 is also update to "microsoft-azure" via https://github.com/MicrosoftDocs/SupportArticles-docs-pr/pull/7550. Thanks.

**Comment by divargas-msft** [comment]
[heart]         Diego Vargas reacted to your message:
________________________________

**Pull request merged**